gql-queries/vol_qry.r

```diff
@@ -0,0 +1,28 @@
+library(glue)
+
+vol_qry <- function(id, from, to) {
+  query_string <- glue(
+    '{
+      trafficData(trafficRegistrationPointId: "{id}") {
```

Comment:
You need to add double double quotes before the {id}. If not it will not be interpreted as a string.

Here are the lines you would need to change:

**Old line:**
```
trafficData(trafficRegistrationPointId: "{id}") {
```

**New line:**
```
trafficData(trafficRegistrationPointId: ""{id}"") {
```

**Old line:**
```
byHour(from: "{from}", to: "{to}") {
```

**New line:**
```
byHour(from: ""{from}"", to: ""{to}"") {
```